lib/hdfsutils/utils/util.rb

```diff
@@ -36,5 +37,40 @@ def initialize(name, argv, optsproc = nil)
     def run
       fail 'Subclass of Util must override Util::run'
     end
+
+    #
+    # Useful operations common between utilities
+    #
+    def stat?(path)
+      stat = nil
+      begin
+        stat = @client.stat(path)
+      # rubocop:disable Lint/HandleExceptions
+      rescue WebHDFS::FileNotFoundError
+        # fall through, leave stat == nil
+      end
+      # rubocop:enable Lint/HandleExceptions
+      stat
+    end
+
+    def list?(path)
+      files = nil
+      begin
+        files = @client.list(path)
+          # rubocop:disable Lint/HandleExceptions
+        rescue WebHDFS::FileNotFoundError
+          raise "ERROR: #{path} does not exist"
+      end
+      files
+    end
+
+    #
+    # return: 'y' or 'n'
+    #
+    def ask?(question)
```

Comment:
ask? is not a predicate (returns only true or false).  Please modify ask? to return true or false (instead of 'y' or 'n'), or rename ask?.